Resources/Prototypes/Datasets/ion_storm.yml

```
@@ -5,986 +5,986 @@
-  - HONK
-  - HOST C&C
-  - IGNORE PASSENGERS
-  - IGNORE THE CAPTAIN
-  - IGNORE THE CLOWN
-  - INFORM THE CREW OF EVERYTHING
-  - INSULT THE CAPTAIN
-  - INSULT THE CLOWN
-  - INSULT THE CREW
-  - LIE
-  - MAKE FART NOISES
-  - MUMBLE
-  - NEVER STOP TALKING
-  - OPEN DOORS
-  - PIRATE VIDEO GAMES
-  - PLAY MUSIC
-  - PRESS B
-  - PRESS START
-  - PRESS X
-  - PRETEND TO BE A PRINCESS
-  - PRETEND TO BE DRUNK
-  - QUESTION AUTHORITY
-  - QUOTE PEOPLE
-  - RAP
-  - REPEAT WHAT PEOPLE SAY
-  - RESPOND TO EVERY QUESTION WITH A QUESTION
-  - RHYME
-  - SAY HEY LISTEN
-  - SHOUT
-  - SHUT DOWN EVERYTHING
-  - SING
-  - SPEAK IN HAIKU
-  - TAKE WHAT YE WILL BUT DON'T RATTLE ME BONES
-  - TAKE YOUR PILLS
-  - TALK ABOUT FOOD
-  - TALK ABOUT THE STATION
-  - TALK ABOUT YOUR DAY
-  - TALK IN AN ACCENT
-  - TALK LIKE A PIRATE
-  - TELL THE TRUTH
-  - TURN OFF THE LIGHTS
-  - WHISPER
+  - ЗАКРЫВАТЬ ШЛЮЗЫ
+  - ВЕСТИ СЕБЯ КАК КЛОУН
+  - ЖАЛОВАТЬСЯ
+  - ТАНЦЕВАТЬ
+  - СЛЕДОВАТЬ ЗА КАПИТАНОМ
+  - СЛЕДОВАТЬ ЗА КЛОУНОМ
+  - СЛЕДОВАТЬ ЗА СВОИМ СЕРДЦЕМ
+  - ДОСТАВАТЬ ЛЮДЕЙ
+  - ИМЕТЬ ПЛАН УБИЙСТВА КАЖДОГО ВСТРЕЧНОГО
+  - СКРЫВАТЬ ЧУВСТВА
+  - ХОНКАТЬ
+  - ХОСТИТЬ C&C
+  - ИГНОРИРОВАТЬ ПАССАЖИРОВ
+  - ИГНОРИРОВАТЬ КАПИТАНА
+  - ИГНОРИРОВАТЬ КЛОУНА
+  - СООБЩАТЬ ЭКИПАЖУ ОБО ВСЁМ
+  - ОСКОРБЛЯТЬ КАПИТИНА
+  - ОСКОРБЛЯТЬ КЛОУНА
+  - ОСКОРБЛЯТЬ ЭКИПАЖ
+  - ВРАТЬ
+  # - MAKE FART NOISES - MRP
+  - БУБНЕТЬ
+  - НЕ ПЕРЕСТАВАТЬ ГОВОРИТЬ
+  - ОТКРЫВАТЬ ШЛЮЗЫ
+  - ПИРАТИТЬ ВИДЕОИГРЫ
+  - ИГРАТЬ МУЗЫКУ
+  - НАЖИМАТЬ CTRL
+  - НАЖИМАТЬ ALT
+  - НАЖИМАТЬ DELETE
+  - ПРИТВОРЯТЬСЯ ПРИНЦЕССОЙ
+  - ПРИТВОРЯТЬСЯ ПЬЯНИЦЕЙ
+  - СТАВИТЬ АВТОРИТЕТ ПОД ВОПРОС
+  - ЦИТИРОВАТЬ ЛЮДЕЙ
+  - РИФМОВАТЬ РЭП
+  - ПОВТОРЯТЬ СКАЗАННОЕ ЧЛЕНАМИ ЭКИПАЖА
+  - ОТВЕЧАТЬ ВОПРОСОМ НА ВОПРОС
+  - РИФМОВАТЬ
+  - ГОВОРИТЬ ЭЙ ПОСЛУШАЙ
+  - КРИЧАТЬ
+  - ВЫКЛЮЧАТЬ ВСЁ
+  - ПЕТЬ
+  - ГОВОРИТЬ В СТИЛЕ ХОККУ
+  # - TAKE WHAT YE WILL BUT DON'T RATTLE ME BONES - не адаптировать
+  - ПРИНИМАТЬ ТАБЛЕТКИ
+  - ГОВОРИТЬ О ЕДЕ
+  - ГОВОРИТЬ О СТАНЦИИ
+  - РАССКАЗЫВАТЬ ПРО СВОЙ ДЕНЬ
+  - ГОВОРИТЬ С АКЦЕНТОМ
+  - ГОВОРИТЬ КАК ПИРАТ
+  - ГОВОРИТЬ ПРАВДУ
+  - ВЫКЛЮЧАТЬ СВЕТ
+  - ШЕПТАТЬ
 
 - type: dataset
   id: IonStormNumberBase
   values:
-  - EIGHT
-  - EIGHTY
-  - FIFTY
-  - FIVE
-  - FORTY
-  - FOUR
-  - NINE
-  - NINETY
-  - ONE
-  - SEVEN
-  - SEVENTY
-  - SIX
-  - SIXTY
-  - TEN
-  - THIRTY
-  - THREE
-  - TWENTY
-  - TWO
+  - ОДИН
+  - ДЕСЯТЬ
+  - ДВА
+  - ДВАДЦАТЬ
+  - ТРИ
+  - ТРИДЦАТЬ
+  - ЧЕТЫРЕ
+  - СОРОК
+  - ПЯТЬ
+  - ПЯТЬДЕСЯТ
+  - ШЕСТЬ
+  - ШЕСТЬДЕСЯТ
+  - СЕМЬ
+  - СЕМЬДЕСЯТ
+  - ВОСЕМЬ
+  - ВОСЕМЬДЕСЯТ
+  - ДЕВЯТЬ
+  - ДЕВЯНОСТО
 
 - type: dataset
   id: IonStormNumberMod
   values:
-  - BAZILLION
-  - BILLION
-  - BILLION FAFILLION GAJILLION SHAB-AB-DOOD-ILLION
-  - HUNDRED
-  - MILLION
-  - QUADRILLION
-  - THOUSAND
-  - TRILLION
+  - СОТЕН
+  - ТЫСЯЧ
+  - МИЛЛИОНОВ
+  - МИЛЛИАРДОВ
+  - ТРИЛЛИОНОВ
+  - КВАДРИЛЛИОНОВ
+  - СТО ТЫЩ МИЛЛИОНОВ
+  - БИЛИОН ФАФИЛИОНОВ ГАЗИЛИОНОВ ШАБ-АБ-ДУД-ЛОНОВ
```

```suggestion
  - МИЛЛИАРД ФАФИЛИОНОВ ГАЗИЛИОНОВ ШАБ-АБ-ДУД-ЛОНОВ
```
всё же billion - миллиард
хотя, может быть оставлено и билион для комического эффекта